brew/brew.go

```diff
@@ -57,13 +57,13 @@ func (c *HomebrewClient) LatestTag(ctx context.Context) (*version.Version, error
 	out, err := pipeline.Output(
 		[]string{c.cmdPath, "info", c.getFullName()},
 		[]string{"head", "-1"},
-		[]string{"awk", "{ print $3 }"},
+		[]string{"awk", "{ print $4 }"},
```

Comment:
It's possible to panic when `brew` command is old. Can you replace awk with grep to extract only the version from the line?

```suggestion
		[]string{"grep", "-o", "-E", "([0-9]+\.){1}[0-9]+(\.[0-9]+)?"},
```

Reply:
@ktr0731 👍